lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,17 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(opts, callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    streamComputationWebsocket(opts, requestId, callbacks);
```

Comment:
@mpetazzoni I added this new method here to register a live tail as we spoke. 

I am confused by what is expected by the opts input argument. Looks like addComputation method which is called by streamComputationWebsocket needs a params.range as per https://github.com/signalfx/signalfx-nodejs/blob/master/lib/client/signalflow/request_manager.js#L233

Also, looks like a requestType and isRetryPatchMode are needed here. What do these arguments mean and what do I provide for these?
 https://github.com/signalfx/signalfx-nodejs/blob/master/lib/client/signalflow/request_manager.js#L259

Reply:
You shouldn't use `streamComputationWebsocket()`, this is specifically for starting a Signalflow computation. You'll want to send your own request into the Websocket (with `activeSocket.send(...)`, and make sure you register something into the `knownComputations` map.

Reply:
Ahh got it. Let me update